### ISSUE TITLE
Child elements of <maintenanceAgency>

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#165]
Make one of the elements `<agencyCode>` or `<agencyName>` mandatory as child element of `<maintenanceAgency>`.

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. silkejagodzinski@gmail.com

## The issue relates to

- [x] EAC-CPF schema issue
- [x] EAC-CPF Tag Library issue
- [ ] EAD schema issue
- [ ] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature
To avoid an empty mandatory wrapper element in `<control>`, make one of the elements `<agencyCode>` or `<agencyName>` mandatory by using xs:choice.

**Attention:** `<agencyCode>` must not be repeatable, we use `<otherAgencyCode>` for that purpose, whereas `<agencyName>` is repeatable for multilingualism.

## Reporting a bug
Not available in this schema version from [15 Sep.](https://github.com/SAA-SDT/eac-cpf-schema/commit/e458b6e21f27f5b4be209085cd53d9b6ebb52923#diff-da3fb4cbcceb766c01c0a8b208dbe8fc50fb348d81c30f661ae97136809c7c66)

**Comment by kerstarno** [comment]
@SJagodzinski - could you please specify if this is mainly reporting a bug, i.e. `<agencyName>` not being required in the XSD draft for EAC-CPF 2.0 (see my last comment and fix in https://github.com/SAA-SDT/eac-cpf-schema/issues/86), or suggesting a new feature, i.e. allowing a choice between `<agencyCode>` and `<agencyName>` and thereby allowing an encoding such as 
```
<maintenanceAgency>
  <agencyCode>FR-1234</agencyCode>
</maintenanceAgency>
```

**Comment by SJagodzinski** [comment]
Solved by [EAC-CPF 2.0 schema release](https://github.com/SAA-SDT/eac-cpf-schema/releases/tag/v2.0-beta): either `<agencyName>` or `<agencyCode>` is required as child of `<maintenanceAgency>`.
